examples/habitat_cl_example.py

```diff
@@ -0,0 +1,90 @@
+from argparse import ArgumentParser
+
+import cv2
+
+from habitat.config import get_multi_task_config
+from habitat.core.env import MultiTaskEnv
+from habitat.sims.habitat_simulator.actions import HabitatSimActions
+from habitat.tasks.nav.nav import NavigationEpisode
+
+FORWARD_KEY = "w"
+LEFT_KEY = "a"
+RIGHT_KEY = "d"
+FINISH = "f"
+
+
+def rgb2bgr(image):
+    return image[:, :, [2, 1, 0]]
```

Comment:
```suggestion
    return image[..., ::-1]
```
or use the cv2.cvtColor function.

Reply:
I didn't want to change too much from https://aihabitat.org/docs/habitat-lab/quickstart.html for clarity. I'll fix it asap.